jwst/assign_wcs/nirspec.py

```diff
@@ -1224,8 +1224,9 @@ def nrs_ifu_wcs(input_model):
                       'nrs_focus': imaging,
                       'nrs_mimf': imaging,
                       'nrs_bota': imaging,
-                      'nrs_autoflat': not_implemented_mode, #TBD
-                      'nrs_autowave': not_implemented_mode, #TBD
+                      'nrs_autoflat': not_implemented_mode,
+                      'nrs_autowave': not_implemented_mode,
                       'nrs_lamp': slits_wcs,
-                      'nrs_brightobj': slits_wcs
+                      'nrs_brightobj': slits_wcs,
+                      'nrs_dark': not_implemented_mode,
```

Comment:
Where's the list for FGS? The valid exp_types for it are:

FGS_IMAGE
FGS_FOCUS (same as imaging)
FGS_SKYFLAT (same as imaging)
FGS_INTFLAT (not_implemented)
FGS_DARK (not_implemented)